libs/core/langchain_core/runnables/history.py

```diff
@@ -28,8 +25,11 @@
     from langchain_core.runnables.config import RunnableConfig
     from langchain_core.tracers.schemas import Run
 
+import inspect
+from typing import Callable, Dict, Union
+
 MessagesOrDictWithMessages = Union[Sequence["BaseMessage"], Dict[str, Any]]
-GetSessionHistoryCallable = Callable[..., BaseChatMessageHistory]
+GetSessionHistoryCallable = Union[Callable[..., BaseChatMessageHistory]]
```

Comment:
Does this do anything?

Reply:
A union of a single element I think is actually invalid